data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+			`	The <ship> settles into the landing bay, and the technician quickly disembarks. Apparently there must have been other shipments while you were gone, as there are now a handful of people visible and quite a bit more equipment stacked everywhere. The station itself feels more alive, the air fresher. As before, you promptly receive notification of another payment of <payment> to your account.`
+
+
+
+mission "Remnant: Cognizance 10"
+	name "Remnant Data Delivery"
+	description "Deliver a datacube to the engineers on Viminal."
+	landing
+	source "Esquiline"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 9: done"
+	on offer
+		conversation
+			`As you finish unloading you spot Gavriil approaching. He gestures in greeting as he approaches. "Thanks for the delivery. We've got standard logistics routes set up now for the rest."  He pauses and fishes a datacube out of a pouch. "Could you take this to Prefect Taely on Viminal? She is overseeing some retrofits and will need this data.`
+				accept
+	on complete
+		conversation
+			`On viminal you easily locate the maintenance yards and eventually find Taely burried in a Pelican. She spots you approaching and frees herself from the machinery. "Greetings <first>. How is our station faring?" You reply that it is improving rapidly, and hand her the datacube. "Ah, good. The detailed chemical analysis and profiles." She slots it into her console and taps a few commands. "OK, I just need to incorporate these profiles into the system and make a few more adjustments. Meet me in the cafeteria in a couple hours."`
+
+
+
+mission "Remnant: Cognizance 11"
+	name "Travel with the <escort target ship> to pick up a shipment of atmosphere from Nasqueron and deliver it to <destination>."
+	description "Go meet up with escorts"
+	cargo equipment 8
+	source "Viminal"
+	stopover "Nasqueron"
+	destination "Esquiline"
+	npc accompany
+		government "Remnant"
+		personality timid launching escort
+		fleet
+			names "remnant"
+			cargo 8 equipment
+			variant
+				Pelican
+				Gascraft 4
+	to offer
+		has "Remnant: Cognizance 10: done"
+	on offer
+		conversation
+			`In the cafeteria you are met by the Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with a cargo of equipment designed to harvest some atmosphere from the void sprite's worlds, and bring it back to Esquiline. The equipment on the gascrafts will collect, compress, and store the gas, and them you can offload it to specialy prepared tanks on the pelican. Then proceed to Esquiline and offload what you have collected to the station.`
```

Comment:
`In the cafeteria, you are met by Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with a cargo of equipment designed to harvest some atmosphere from the void sprite's worlds before bringing it back to Esquiline. The equipment on the gascrafts will collect, compress, and store the gas, which you can offload to specially prepared tanks on the pelican. Then proceed to Esquiline and offload what you have collected to the station.`

Reply:
Changed.